eks-s3-seldon-mlflow/README.md

```diff
@@ -1,6 +1,6 @@
-# EKS, S3, Seldon and MLflow Stack 
+# EKS, S3, RDS, Seldon and MLflow Stack 
```

Comment:
Perhaps we are missing a motivation of this stack. When should it be used? Why is it like the way it is? Etc.